htdocs/helpFiles/Entering-Syntax.html

```diff
@@ -1,137 +1,249 @@
-sint or sin t even though WeBWorK is smart enough to <b>usually</b> accept sin t or even sint. For example, sin 2t is interpreted as sin(2)t, i.e. (sin(2))*t so be careful.
-
-</li><li> You can enter sin^2(t) as a short cut although mathematically
-speaking sin^2(t) is shorthand for (sin(t))^2(the square of sin of t).
-(You can enter it as sin(t)^2 or even sint^2, but don't try such things
-unless you <b>really</b> understand the precedence of operations. The
-"sin" operation has highest precedence, so it is performed first, using
-the next token (i.e. t) as an argument. Then the result is squared.)
-You can always use the Preview button to see a typeset version of what
-you entered and check whether what you wrote was what you
-meant.&nbsp;:-)
-</li><li> For example 2+3sin^2(4x) will work and is equivalent to
-2+3(sin(4x))^2 or 2+3sin(4x)^2. Why does the last expression work?
-Because things in parentheses are always done first [ i.e. (4x)], next
-all functions, such as sin, are evaluated [giving sin(4x)], next all
-exponents are taken [giving sin(4x)^2], next all multiplications and
-divisions are performed in order from left to right [giving
-3sin(4x)^2], and finally all additions and subtractions are performed
-[giving 2+3sin(4x)^2].
-</li><li> Is -5^2 positive or negative? It's negative. This is because
-the square operation is done before the negative sign is applied. Use
-(-5)^2 if you want to square negative 5.
-</li><li> When in doubt use parentheses!!!&nbsp;:-)
-</li><li> The complete rules for the precedence of operations, in addition to the above, are
-<ul><li> Multiplications and divisions are performed left to right: 2/3*4 = (2/3)*4 = 8/3.
-
-</li><li> Additions and subtractions are performed left to right: 1-2+3 = (1-2)+3 = 2.
-</li><li> Exponents are taken right to left: 2^3^4 = 2^(3^4) = 2^81 = a big number.
-</li></ul>
-</li><li> <b>Use the "Preview Button" to see exactly how your entry
-looks. E.g. to tell the difference between 1+2/3*4 and [1+2]/[3*4]
-click the "Preview Button".</b>
-</li></ul>
-<a name="Mathematical_Constants_Available_In_WeBWorK"></a><h4> <span class="mw-headline">Mathematical Constants Available In WeBWorK</span></h4>
-<ul><li> pi This gives 3.14159265358979, e.g. cos(pi) is -1
-</li><li> e This gives 2.71828182845905, e.g. ln(e*2) is 1 + ln(2)
-</li></ul>
-
-<a name="Scientific_Notation_Available_In_WeBWorK"></a><h4> <span class="mw-headline">Scientific Notation Available In WeBWorK</span></h4>
-<ul><li> 2.1E2 is the same as  210
-</li><li> 2.1E-2 is the same as .021
-</li></ul>
-<a name="Mathematical_Functions_Available_In_WeBWorK"></a><h4> <span class="mw-headline">Mathematical Functions Available In WeBWorK</span></h4>
-<p>Note that sometimes one or more of these functions is disabled for a WeBWorK problem because the 
-instructor wants you to calculate the answer by some means other than just using the function.
+<h2>Syntax for entering answers to WeBWorK</h2>
```

Comment:
For consistency, should this be:
```
<h2 style="text-align: center">Syntax for entering answers to WeBWorK</h2>
```

Reply:
I missed that.  Thanks.  I added that now.

However, should I also center the other headings in that file?  It looks a bit off with only the top heading centered.  That file (and its duplicate) are the only help files that have additional headings.  Wait, I see that the PDE-notation.html file does also.